gpu-aware-scheduling/docs/usage.md

```diff
@@ -65,6 +65,61 @@ Your PODs then, needs to ask for some GPU-resources. Like this:
 
 A complete example pod yaml is located in [docs/example](./example)
 
+## Node Label support
+
+GAS supports certain node labels as a means to allow telemetry based GPU selection decisions and
+descheduling of PODs using a certain GPU. You can create node labels with the
+[Telemetry Aware Scheduling](../../telemetry-aware-scheduling/README.md) labeling strategy,
+which puts them in its own namespace. In practice the supported labels need to be in the
+`telemetry.aware.scheduling.POLICYNAME/` namespace, where the POLICYNAME may be anything.
+
+The node label `gas-deschedule-pods-GPUNAME` where the GPUNAME can be e.g. card0, card1, card2...
+which corresponds to the gpu names under /dev/dri, will result in GAS labeling the PODs which
+use the named GPU with the `gpu.aware.scheduling/deschedule-pod=gpu` label. You may then
+use with a kubernetes descheduler to pick the pods for descheduling. So TAS labels the node, and
+based on the node label GAS finds and labels the PODs. Descheduler can be configured to
+deschedule the pods based on pod labels.
+
+The node label `gas-disable-GPUNAME` where the GPUNAME can be e.g. card0, card1, card2... which
+corresponds to the gpu names under /dev/dri, will result in GAS stopping the use of the named
+GPU for new allocations.
+
+The node label `gas-prefer-gpu=GPUNAME` where the GPUNAME can be e.g. card0, card1, card2...
```

Comment:
Is it gas-prefer-gpu=GPUNAME or gas-prefer-gpu-GPUNAME. Seems like a typo unless you intend to have the gpu name as the label's value.

Reply:
The intention is to have the gpu name as the label's value. Here the TAS labeling strategy can be used to get the best metrics value producing GPU to have the value field filled up correctly. For that, the label key must match.